common/src/main/java/org/jboss/hal/testsuite/page/config/ApplicationSecurityDomainPage.java

```diff
@@ -0,0 +1,51 @@
+package org.jboss.hal.testsuite.page.config;
+
+import org.jboss.arquillian.graphene.Graphene;
+import org.jboss.arquillian.graphene.findby.ByJQuery;
+import org.jboss.hal.testsuite.finder.Application;
+import org.jboss.hal.testsuite.finder.FinderNames;
+import org.jboss.hal.testsuite.fragment.ConfigAreaFragment;
+import org.jboss.hal.testsuite.fragment.ConfigFragment;
+import org.jboss.hal.testsuite.fragment.config.undertow.AddApplicationSecurityDomainWizard;
+import org.jboss.hal.testsuite.fragment.config.undertow.AddSSOWizard;
+import org.jboss.hal.testsuite.util.Console;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
```

Comment:
Please add javadoc - at least that this is page for Undertow application security domain configuration (yes it's clear from the navigation method but should be in javadoc as well IMHO)